docs/examples/flight_replay.rst

```diff
@@ -2,57 +2,95 @@
-	Sent waypoint 0 : MISSION_ITEM {target_system : 1, target_component : 1, seq : 0, frame : 3, command : 16, current : 0, autocontinue : 0, param1 : 0, param2 : 0, param3 : 0, param4 : 0, x : 45.7379052, y : 126.6273574, z : 30.0}
-	Sent waypoint 1 : MISSION_ITEM {target_system : 1, target_component : 1, seq : 1, frame : 3, command : 16, current : 0, autocontinue : 0, param1 : 0, param2 : 0, param3 : 0, param4 : 0, x : 45.7378905, y : 126.6273609, z : 30.0}
-	Sent waypoint 2 : MISSION_ITEM {target_system : 1, target_component : 1, seq : 2, frame : 3, command : 16, current : 0, autocontinue : 0, param1 : 0, param2 : 0, param3 : 0,
-	...
-	Sent waypoint 92 : MISSION_ITEM {target_system : 1, target_component : 1, seq : 92, frame : 3, command : 16, current : 0, autocontinue : 0, param1 : 0, param2 : 0, param3 : 0, param4 : 0, x : 45.737971, y : 126.6274908, z : 30.0}
-	Sent waypoint 93 : MISSION_ITEM {target_system : 1, target_component : 1, seq : 93, frame : 3, command : 16, current : 0, autocontinue : 0, param1 : 0, param2 : 0, param3 : 0, param4 : 0, x : 45.738018, y : 126.6275664, z : 30.0}
-	Sent waypoint 94 : MISSION_ITEM {target_system : 1, target_component : 1, seq : 94, frame : 3, command : 16, current : 0, autocontinue : 0, param1 : 0, param2 : 0, param3 : 0, param4 : 0, x : 45.7380429, y : 126.6275067, z : 30.0}
-	Sent all 95 waypoints
-	Got MAVLink msg: MISSION_ACK {target_system : 255, target_component : 0, type : 0}
-	APM: flight plan received
+Running the example
+===================
+
+The example can be run as described in :doc:`running_examples` (which in turn assumes that the vehicle
+and DroneKit have been set up as described in :ref:`get-started`). 
+
+If you're using a simulated vehicle, remember to :ref:`disable arming checks <disable-arming-checks>` so 
+that the example can run.
+
+In summary, after cloning the repository:
+
+#. Navigate to the example folder as shown:
+
+   .. code-block:: bash
+
+       cd dronekit-python\examples\flight_replay\
+
+
+#. Start the example, passing the :ref:`connection string <get_started_connect_string>` 
+   you wish to use in the ``--connect`` parameter and specifying the mission to replay.
+
+   .. code-block:: bash
+
+       python flight_replay.py --connect 127.0.0.1:14550 --mission_id 101
+
+   .. note::
+   
+       The command parameters above are the defaults, and may be omitted. These
+       connect to SITL on udp port 127.0.0.1:14550 and replay the mission with id 101.
+
+       
+.. tip::
+
+    It is more interesting to watch the example above on a map than the console. The topic :ref:`viewing_uav_on_map` 
+    explains how to set up *Mission Planner* to view a vehicle running on the simulator (SITL).
+
+On the command prompt you should see (something like):
+
+.. code-block:: bash
+
+
+    \dronekit-python\examples\flight_replay>flight_replay.py
+    Connecting to vehicle on: 127.0.0.1:14550
+    >>> ☺APM:Copter V3.4-dev (e0810c2e)
```

Comment:
Can you remove these smiley faces? They're just a bug on master :) Was an errant, random character.